src/cryptography/x509/extensions.py

```diff
@@ -76,8 +75,8 @@ def _key_identifier_from_public_key(public_key: _PUBLIC_KEY_TYPES) -> bytes:
     return hashlib.sha1(data).digest()
 
 
-def _make_sequence_methods(field_name):
-    def len_method(self):
+def _make_sequence_methods(field_name: str):
+    def len_method(self) -> int:
```

Comment:
This function is basically untypable, I think. Open to suggestions, of course.

Note that here `self` is not even bound, so even that doesn't check out. `iter_method` below would have to return `Iterator[WHAT?]`. And `_make_sequence_methods` would have to be returned with several callables, all with a type.

This could be done with a generic typevar. But this will add a lot of `typing.TypeVar` variables. Is this really what we want? Alternatively, we can also just implement this function, "the long way around", like I did with `Extensions` below.